src/makeRobotAccountant.js

```diff
@@ -6,7 +6,18 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+//   // write code here
+  let count = 0;
+
+  return function(a) {
+    return function(b) {
```

Comment:
You can add 0 as a default value. So if no argument is passed, it would still work. But your code works fine.

Reply:
agree with @ysrckr also setting default values to 0 is a good idea